### PR TITLE
update the help and readme

### PR DESCRIPTION
**Pull request by lbergelson**
updating the help and readme to be more useful and up to date

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,25 +5,63 @@
...
+     * *`daemon-server`*  server to run the notification daemon on
+
+   #### Display a list jobs submitted through cromshell
+   * `list` *`[-c]` `[-u]`*                                            
+     * *`-c`*    Color the output by completion status
+     * *`-u`*    Check completion status of all unfinished jobs
    
  ### Features:
- * Running `submit` will create a new folder in your working directory named with the cromwell job id of the newly submitted job.  
+ * Running `submit` will create a new folder in your `~/.cromshell` directory named with the cromwell job id of the newly submitted job.
```
Comment (jonn-smith) [inline README.md]: It actually creates a folder for a particular server, then a sub-folder for the job ID

File: README.md [hunk 2]
```diff
@@ -5,25 +5,63 @@
...
+ * Running `submit` will create a new folder in your `~/.cromshell` directory named with the cromwell job id of the newly submitted job.  
  It will copy your wdl and json inputs into the folder for reproducibility.  
  * It keeps track of your most recently submitted jobs by storing their ids in `./cromshell/`  
  You may ommit the job ID of the last job submitted when running commands, or use negative numbers to reference previous jobs, e.g. "-1" will track the last job, "-2" will track the one before that, and so on.
- * You can override the default cromwell server by setting the environmental variable CROMWELL_URL to the appropriate URL.
- * The commands `status` and `abort` takes multiple workflow-id's, but only accepts the explicit ids when multiple ids are provided, i.e. `./cromwell status -1 -2 -3` won't work.
+ * You can override the default cromwell server by setting the environmental variable `CROMWELL_URL` to the appropriate URL.
+ * Most commands takes multiple workflow-id's, but only accepts the explicit ids when multiple ids are provided, i.e. `./cromwell status -1 -2 -3` won't work.
```
Comment (jonn-smith) [inline README.md]: We can add a PR to fix this if you like.

Comment (lbergelson) [reply README.md]: It would be nice to fix but very low priority.

File: README.md (hunk 1 above)
Comment (lbergelson) [reply README.md]:
```suggestion
 * Running `submit` will create a new folder in the `~/.cromshell/${CROMWELL_URL}/` directory named with the cromwell job id of the newly submitted job.  
```

**Pull request merged**